docs/i18n.md

```diff
@@ -131,3 +131,16 @@ import { Trans } from 'react-i18next';
 
 // ...
 ```
+
+
```

Comment:
Can you add "Enabling translations" or similar title here? Now this is under "Translation with variables". ;-)